.tools/autoinstall

```diff
@@ -114,6 +114,66 @@ function install_auditd() {
   esac
 }
 
+function install_laurel() {
+  display_installation_step "Installing Laurel"
+
+  curl -LOs https://github.com/quantum-sec/laurel/releases/download/v0.3.1/laurel-v0.3.1-x86_64-musl.tar.gz
+  trap 'rm laurel-v0.3.1-x86_64-musl.tar.gz' EXIT
+  sudo tar -zxf laurel-v0.3.1-x86_64-musl.tar.gz -C /usr/local/sbin laurel
+  if ! id -u "_laurel" >/dev/null 2>&1; then
+    sudo useradd --system --home-dir /var/log/laurel --create-home _laurel
+  fi
+}
+
+function configure_laurel() {
+  display_installation_step "Configuring Laurel"
+
+  readonly LAUREL_CONFIG_FILE='/etc/laurel/config.toml'
+  sudo mkdir /etc/laurel
+  curl -s https://raw.githubusercontent.com/quantum-sec/laurel/master/etc/laurel/config.toml \
+  | sudo tee "$LAUREL_CONFIG_FILE" > /dev/null
+  sudo sed -i "s/\(read-users *= *\).*/\1\[ \"$(whoami)\" \]/" "$LAUREL_CONFIG_FILE"
+
+  case "$OS_NAME" in
+  Ubuntu)
+    AUDITD_PLUGIN_DIR='/etc/audisp/plugins.d'
+    trap 'sudo chmod 640 $LAUREL_CONFIG_FILE "$AUDITD_PLUGIN_DIR"/laurel.conf; trap - RETURN' RETURN
+    curl -s https://raw.githubusercontent.com/quantum-sec/laurel/master/etc/audit/plugins.d/laurel.conf \
+    | sudo tee "$AUDITD_PLUGIN_DIR"/laurel.conf > /dev/null
+    sudo sed -i '/#dispatcher/s/^#//g' "$AUDITD_CONFIG_FILE"
+    ;;
+  CentOS)
+    AUDITD_PLUGIN_DIR='/etc/audit/plugins.d'
+    trap 'sudo chmod 640 $LAUREL_CONFIG_FILE "$AUDITD_PLUGIN_DIR"/laurel.conf; trap - RETURN' RETURN
+    curl -s https://raw.githubusercontent.com/quantum-sec/laurel/master/etc/audit/plugins.d/laurel.conf \
+    | sudo tee "$AUDITD_PLUGIN_DIR"/laurel.conf > /dev/null
+    sudo sed -i '/#plugin_dir/s/^#//g' "$AUDITD_CONFIG_FILE"
+    ;;
+  esac
+}
+
+function configure_laurel_with_os_specific_config() {
+  #SeLinux
+  case "$OS_NAME" in
```

Comment:
Maybe we can add a catch (default) case that if its not ubuntu or centos it would give an error of unsupported os

Reply:
Sure, Let me add that

Reply:
Oh, i think @GerardSetho has already added that check
https://github.com/quantum-sec/auditd-config/blob/580797612de8089df8a56148eaff4e10f2477cda/.tools/autoinstall#L38-L40